src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -190,4 +196,43 @@ public void setAuthorAliases(Author author, String... aliases) {
             authorAliasMap.put(alias, author);
         }
     }
+
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    public String getLocation() {
+        return location;
+    }
+
+    public String getRepoName() {
+        return repoName;
+    }
+
+    /**
+     * Verifies {@code location} can be presented as a {@code URL} or {@code Path}.
+     * @throws ParseException if otherwise.
+     */
+    private void verifyLocation(String location) throws ParseException {
+        boolean isPathLocation = false;
+        boolean isGitLocation = false;
+
+        try {
+            Path pathLocation = Paths.get(location);
+            isPathLocation = Files.exists(pathLocation);
```

Comment:
similarly, maybe `isValidFilePath`? Also, if we're standardizing with url, perhaps you want to add `&& location.endsWith(GIT_LINK_SUFFIX);`